cmd/snap/cmd_debug_api.go

```diff
@@ -67,13 +70,19 @@ func init() {
 		func() flags.Commander {
 			return &cmdDebugAPI{}
 		}, map[string]string{
-			"header":  "Set header (can be repeated multiple times), header kind and value are separated with ': '",
-			"request": "HTTP method to use (defaults to GET)",
-			"fail":    "Fail on request errors",
+			"header":      "Set header (can be repeated multiple times), header kind and value are separated with ': '",
+			"request":     "HTTP method to use (defaults to GET)",
+			"fail":        "Fail on request errors",
+			"snap-socket": "Use snap access socket",
```

Comment:
This is vague, can you clarify that this is using the socket used by snapctl when SNAP_NAME is set?

Reply:
hm but it's using the snap socket only when `--snap-socket` is passed in the command line, SNAP_NAME is not checked at all